index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    
+    this.items.push(item);
 
-  get(pos) {}
+    this.items.sort((a,b) => {
+      if( a < b ) return -1;
+      if( a > b ) return 1;
+      if( a === b ) return 0;
+    })
 
-  max() {}
+    this.length = this.items.length
 
-  min() {}
+  }
 
-  sum() {}
+  get(pos) {
 
-  avg() {}
+    if(this.items[pos] === undefined) throw new Error("OutOfBounds")
```

Comment:
```suggestion
    if(this.items[pos] === undefined) {
        throw new Error("OutOfBounds")
    }
```
Esta perfect como lo has puesto eeee, pero en cleanCode especifican que es mejor poner las llaves vale? solo por eso te lo digo, son buenas practicas! :D